### PR TITLE
handle FileNotFoundError in parse_requirements function

### PR DESCRIPTION
**Pull request by mateuslatrova**
When the function "parse_requirements.txt" does not find the file passed as a parameter, it just raises an OSError.

This PR changes that behavior. Now, when that happens, this function handles it by printing a helpful message to the user and ends the program execution immediately.

### CODE REVIEW

**Comment by fredgrub** [comment]
This PR is related to PR https://github.com/bndr/pipreqs/pull/392. Since this PR modifies the functioning of the 'parse_requirements' function, we must accept it first.

**Comment by alan-barzilay** [comment]
LGTM just remove the white trailing space please

**Comment by alan-barzilay** [comment]
ah, and rebase, thx

**Comment by mateuslatrova** [comment]
@alan-barzilay rebased and fixed the white trailing space.

File: tests/test_pipreqs.py [hunk 1]
```diff
@@ -471,6 +457,22 @@ def test_output_requirements(self):
             file_content = f.read().lower()
             stdout_content = capturedOutput.getvalue().lower()
             self.assertTrue(file_content == stdout_content)
+    @patch("sys.exit")
```
Comment (alan-barzilay) [inline tests/test_pipreqs.py:460]: reviewdog is complaining here

File: pipreqs/pipreqs.py [hunk 2]
```diff
@@ -155,56 +151,48 @@ def filter_line(line):
...
-        out_file.write('\n'.join(
-            fmt.format(**item) if item['version'] else '{name}'.format(**item)
-            for item in imports) + '\n')
+        logging.debug(
+            "Writing {num} requirements: {imports} to {file}".format(
+                num=len(imports), file=path, imports=", ".join([x["name"] for x in imports])
+            )
+        )
+        fmt = "{name}" + symbol + "{version}"
+        out_file.write(
```
Comment (alan-barzilay) [inline pipreqs/pipreqs.py:160]: please stop changing formatting and quotes convention on every commit you guys do haha

**Review by alan-barzilay — CHANGES_REQUESTED** [review]

File: tests/test_pipreqs.py (hunk 1 above)
Comment (mateuslatrova) [reply tests/test_pipreqs.py:545]: solved

**Review by alan-barzilay — APPROVED** [review]

**Pull request merged**